samples/cpp/stereo_calib.cpp

```diff
@@ -99,8 +114,29 @@ StereoCalib(const vector<string>& imagelist, Size boardSize, float squareSize, b
                     timg = img;
                 else
                     resize(img, timg, Size(), scale, scale, INTER_LINEAR_EXACT);
-                found = findChessboardCorners(timg, boardSize, corners,
-                    CALIB_CB_ADAPTIVE_THRESH | CALIB_CB_NORMALIZE_IMAGE);
+
+                if (type == "chessboard") {
+                    found = findChessboardCorners(timg, boardSize, corners,
+                        CALIB_CB_ADAPTIVE_THRESH | CALIB_CB_NORMALIZE_IMAGE);
+                }
+                else {
+                    cv::aruco::CharucoBoard ch_board({ boardSize.width + 1, boardSize.height + 1 },
```

Comment:
Why +1?

Reply:
For input pattern size parameters we use number of inner corners. To create ChArUco board we use number of checkers.

Reply:
Charuco and chessboard of the same physical  size and layout should have the same parameters. I propose to use w and h as is, but mention the difference in gen_pattern.py documentation. Otherwise it confuses sample code users.

Reply:
Discussed the issue offline. Decided to remove +1, but add note about the board size units. Chessboard and radon board size are defined in inner corners, all other boards - in elements (circles or squares).